### PR TITLE
[NCL-8063]: Update DB entities

### PR DESCRIPTION
**Pull request by akridI**

### CODE REVIEW

File: src/main/java/org/jboss/pnc/api/enums/DeliverableAnalyzerReportLabel.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    /**
+     * Used to denote that deliverables of the analysis assigned to this report were deleted.
+     * <p>
+     * Note: Adding this label discards the RELEASED label.
+     * </p>
+     */
+    DELETED,
+
+    /**
+     * Used to denote that the deliverable analysis assigned to this report was run as "scratched", perhaps for testing
```
Comment (janinko) [inline src/main/java/org/jboss/pnc/api/enums/DeliverableAnalyzerReportLabel.java:36]: Scratch

File: src/main/java/org/jboss/pnc/api/enums/LabelOperation.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jboss.pnc.api.enums;
+
+/**
+ * LabelOperation labels are used for tracking the history of {@link DeliverableAnalyzerReportLabel}s associated with
```
Comment (janinko) [inline src/main/java/org/jboss/pnc/api/enums/LabelOperation.java:21]: In future it will be for other labels too

**Review by janinko — CHANGES_REQUESTED** [review]

File: src/main/java/org/jboss/pnc/api/enums/LabelOperation.java (hunk 2 above)
Comment (akridI) [reply src/main/java/org/jboss/pnc/api/enums/LabelOperation.java]: done, see 9be5243

File: src/main/java/org/jboss/pnc/api/enums/DeliverableAnalyzerReportLabel.java (hunk 1 above)
Comment (akridI) [reply src/main/java/org/jboss/pnc/api/enums/DeliverableAnalyzerReportLabel.java]: done, see 9be5243

File: src/main/java/org/jboss/pnc/api/enums/DeliverableAnalyzerReportLabel.java [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+/**
+ * DeliverableAnalyzerReportLabel labels are used to define additional metadata about the DeliverableAnalyzerReport.
+ *
+ * @author Adam Kridl &lt;akridl@redhat.com&gt;
+ */
+public enum DeliverableAnalyzerReportLabel {
+
+    /**
+     * Used to denote that deliverables of the analysis assigned to this report were deleted.
```
[the remark this answers is not in the document]
Comment (akridI) [reply src/main/java/org/jboss/pnc/api/enums/DeliverableAnalyzerReportLabel.java]: done, see 9be5243

File: src/main/java/org/jboss/pnc/api/enums/LabelOperation.java [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+/**
+ * LabelOperation labels are used for tracking the history of labels associated with deliverable analyzer label entry.
+ *
+ * @author Adam Kridl &lt;akridl@redhat.com&gt;
+ */
+public enum LabelOperation {
+
+    /**
+     * Used when new {@link DeliverableAnalyzerReportLabel} was added to the report.
```
Comment (janinko) [inline src/main/java/org/jboss/pnc/api/enums/LabelOperation.java:28]: still mentioning DARL

Comment (akridI) [reply src/main/java/org/jboss/pnc/api/enums/LabelOperation.java]: resolved, see fe0463b

**Review by janinko — APPROVED** [review]

**Pull request merged**